docker-compose.yaml

```diff
@@ -10,8 +10,10 @@ services:
       - 443:443
       - 465:465
       - 993:993
-      
+
   james:
+    build:
+      context: local
```

Comment:
For my information: what is this for?

Reply:
You can instruct docker-compose to build your image as part of the docker-compose up. 
https://docs.docker.com/compose/compose-file/#build

Reply:
Also see `docker-compose --help` and `docker-compose up --help` for how to use that when you have a running service.